lib_xua/host/xmosdfu/xmosdfu.cpp

```diff
@@ -66,6 +69,20 @@ unsigned int XMOS_DFU_IF = 0;
 #define XMOS_DFU_SAVESTATE            0xf5
 #define XMOS_DFU_RESTORESTATE         0xf6
 
+enum dfu_state {
```

Comment:
most of these are not used, but I guess that doesn't matter.

Reply:
I copied this from dfu-util. Leaving the unused states here in case we ever need to use them